man/systemd.unit.xml

```diff
@@ -1199,9 +1220,101 @@
+            <entry><varname>Requires=</varname></entry>
+            <entry><varname>RequiredBy=</varname></entry>
+          </row>
+          <row>
+            <entry><varname>Wants=</varname></entry>
+            <entry><varname>WantedBy=</varname></entry>
+          </row>
+          <row>
+            <entry><varname>PartOf=</varname></entry>
+            <entry><varname>ConsistsOf=</varname></entry>
+          </row>
+          <row>
+            <entry><varname>BindsTo=</varname></entry>
+            <entry><varname>BoundBy=</varname></entry>
+          </row>
+          <row>
+            <entry><varname>Requisite=</varname></entry>
+            <entry><varname>RequisiteOf=</varname></entry>
+          </row>
+          <row>
+            <entry><varname>Triggers=</varname></entry>
+            <entry><varname>TriggeredBy=</varname></entry>
+          </row>
+          <row>
+            <entry><varname>Conflicts=</varname></entry>
+            <entry><varname>ConflictedBy=</varname></entry>
+          </row>
+          <row>
+            <entry><varname>PropagatesReloadTo=</varname></entry>
+            <entry><varname>ReloadPropagatedFrom=</varname></entry>
+          </row>
+          <row>
+            <entry><varname>References=</varname></entry>
+            <entry><varname>References=</varname></entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </table>
 
+    <para>Note: <varname>WantedBy=</varname> is used in the [Install] section to
+    create symlinks in <filename>.wants</filename> directories. It cannot be
+    used directly as a unit configuration setting.</para>
+
+    <para>Note: <varname>ConsistsOf=</varname>, <varname>BoundBy=</varname>,
+    <varname>RequisiteOf=</varname>, <varname>ConflictedBy=</varname> are created
+    implicitly along with their reverse and cannot be specified directly.</para>
+
+    <para>Note: <varname>References=</varname> is created implicitly between all
+    units that reference one another in some way and cannot be specified directly.
+    </para>
+
+    <para>Note: <varname>Triggers=</varname> is created implicitly between a socket
+    or path unit and the unit it activates. By default a unit with the same name is
+    triggered, but this can be overriden using <varname>Sockets=</varname>,
+    <varname>Service=</varname>, and <varname>Unit=</varname> settings. See
+    <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+    <citerefentry><refentrytitle>systemd.socket</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+    and <citerefentry><refentrytitle>systemd.path</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+    for details. <varname>TriggersBy=</varname> is created implicitly on the
+    triggered unit.</para>
+
+    <para>Note: <varname>References=</varname> is created implicitly between all
+    units that reference one another in some way and cannot be specified directly.
+    </para>
```

Comment:
this paragraph is already in line 1301

Reply:
Oops, I wanted to move the paragraph, not copy it ;)
Updated.